src/python/WMCore/MicroService/MSOutput/MSOutput.py

```diff
@@ -594,7 +596,9 @@ def msOutputProducer(self, requestRecords):
                 msg = "%s General Error from pipeline. Err: %s. " % (pipeLineName, str(ex))
                 msg += "Giving up Now."
                 self.logger.exception(str(ex))
-                break
+                workflowname = request.get("_id", "none")
```

Comment:
This string `"none"` might be confusing at some point. I would suggest to either set it to the `None` object, or to set it to an empty string `""`.

Reply:
oook